packages/smooth_app/lib/views/smooth_product_sneak_peek_view.dart

```diff
@@ -18,8 +18,10 @@ import 'package:smooth_app/cards/data_cards/smooth_energy_card.dart';
 import 'package:smooth_app/cards/data_cards/smooth_intake_recommendation_card.dart';
 import 'package:smooth_app/cards/data_cards/smooth_quantity_selector_card.dart';
 import 'package:smooth_app/data_models/sneak_peek_model.dart';
-import 'package:smooth_app/data_models/user_preferences_model.dart';
+import 'package:smooth_app/temp/product_preferences.dart';
 
+// TODO(stephanegigandet): remove if not useful anymore?
```

Comment:
in fact I don't know what that code was used for?

Reply:
I believe it used to be something like `ProductPage`.